app/generate_csv_eventyay.py

```diff
@@ -0,0 +1,49 @@
+import urllib.request, json, os
+
+APP_ROOT = os.path.dirname(os.path.abspath(__file__))
+UPLOAD_FOLDER = os.path.join(APP_ROOT, 'static/uploads')
+MAX_STRING = "chaftsladen Potsdam (F"
+
+def tocsv(url_eventyay):
+    with urllib.request.urlopen(url_eventyay) as url:
+        data = json.loads(url.read().decode(url.info().get_param('charset') or 'utf8'))
+        speakers = data['speakers']
+        filename = "speaker.png.csv"
```

Comment:
This could be an argument to the function.

Reply:
The eventyay, for now, provide details of speakers only that's why I didn't pass it as an argument.